lisa/notifier.py

```diff
@@ -40,6 +42,48 @@ class TestRunMessage(MessageBase):
     message: str = ""
 
 
+@dataclass
+class PerfMessage(MessageBase):
+    pass
+
+
+class FIOResult:
+    qdepth: int = 0
+    mode: str = ""
+    iops: Decimal = Decimal(0)
+    latency: Decimal = Decimal(0)
+    iodepth: int = 0
+
+
+@dataclass
+class FIOResultsMessage(PerfMessage):
```

Comment:
It's not limited to FIO. Let's call it like `DiskPerformanceMessage`. Add one more field "tool", and its value is "fio" (a constant).

Reply:
updated